tests/time_tests/test_runner/utils.py

```diff
@@ -150,3 +151,27 @@ def get_os_version():
     if os_type_is_darwin():
         return tuple(platform.mac_ver()[0].split(".")[:2])
     raise UnsupportedOsError()
+
+
+def get_cpu_info():
+    """
+    Check OS version and returns name and frequency of cpu
+
+    :return: CPU name, CPU frequency
+    :rtype: str, str
+    """
+    model = ''
+    if os_type_is_linux():
+        command = r"lscpu | sed -n 's/Model name:[ \t]*//p'"
+        model = subprocess.check_output(command, shell=True)
```

Comment:
Would it be possible to run those commands with shell=False?

something like `re.match(subprocess.check_output('lscpu'), "Model name: (.*)$").match[1]`